src/Services/GTT/GTT.Api/ExcerciseGroupManagement.cs

```diff
@@ -33,6 +33,7 @@ public ExcerciseGroupManagement(ILoggerFactory loggerFactory, IMediator mediator
         [Function("CreateExcerciseGroup")]
         [OpenApiOperation(nameof(ExcerciseGroupManagement), "Excercise")]
         [OpenApiRequestBody(contentType: "application/json", bodyType: typeof(ExGroupRequestModel), Required = true)]
+        [OpenApiResponseWithoutBody(HttpStatusCode.InternalServerError, Description = "Internal Server Error.")]
         public async Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = Routes.CreateExGroup)] HttpRequestData req)
```

Comment:
add 2 more OpenApiResponseWithBody with HttpStatusCode.OK and HttpStatusCode.BadRequest